workflows/ogg_to_mp3.py

```diff
@@ -0,0 +1,50 @@
+import os
+from multiprocessing import Pool
+from pathlib import Path
+
+import luigi
+from google.cloud import storage
+from luigi.parameter import ParameterVisibility
+from pydub import AudioSegment
+
+
+class OggToMP3(luigi.Task):
+    input_path = luigi.Parameter()
+    output_path = luigi.Parameter()
+    parallelism = luigi.IntParameter(default=os.cpu_count(), significant=False)
+
+    def output(self):
+        return luigi.LocalTarget(self.output_path)
+
+    def list_images(self):
```

Comment:
You make reference to images throughout, it might be better to refer them to tracks.